documentation/src/docs/controller-service.md

```diff
@@ -455,337 +414,305 @@ to listen for cluster change notifications. These notify about the added and rem
 across all available Controller instances. To achieve this we rely on
-bucketing streams into predefined sets and distributing these sets
+bucketing Streams into predefined sets and distributing these sets
 across Controller instances. This is done by using Zookeeper to store
 this distribution. Each Controller instance, during bootstrap, attempts
-to acquire ownership of buckets. All streams under a bucket are
+to acquire ownership of buckets. All Streams under a bucket are
 monitored for retention opportunities by the owning Controller. At each
-period, Controller collects a new stream cut and adds it to a
-retention set for the said stream. Post this it looks for candidate
-StreamCuts stored in retention set which are eligible for truncation
-based on the defined retention policy. For example, in time based
-retention, the latest StreamCut older than specified retention period
+period, Controller collects a new `StreamCut` and adds it to a
+retention set for the said Stream. Post this it looks for the candidate
+`StreamCut`(s) stored in retention set which are eligible for truncation
+based on the defined retention policy. For example, in time-based
+retention, the latest `StreamCut` older than the specified retention period
 is chosen as the truncation point.
 
 ## Transaction Manager
 
-Another important role played by Controller is that of Transaction
-manager. It is responsible for creation and committing and abortion of
-transactions. Since Controller is the central brain and agency in our
-cluster, and is the holder of truth about stream, the writers request
-Controller to perform all control plane actions with respect to
-Transactions. Controller plays active roles in providing guarantees for
-Transactions from the time since they are created till the time they are
-committed or aborted. Controller tracks each Transaction for their
-specified timeouts and if the timeout exceeds, it automatically aborts
-the transaction.
-
-Controller is responsible for ensuring that the Transaction and a
+Another important role played by the Controller is that of the Transaction manager. It is responsible for the beginning and
+ending Transactions. The Controller plays an active role in providing guarantees for Transactions from the time
+they are created until the time they are committed or aborted. The Controller tracks each Transaction for their
+specified timeouts, and automatically aborts the Transaction if the timeout exceeds.
+The Controller is responsible for ensuring that the Transaction and a
 potential concurrent scale operation play well with each other and
 ensure all promises made with respect to either are honored and
 enforced.
-
-![transaction management](img/TransactionManagement.png)
-
-Transaction Management
+<p>
+<img src="img/Transaction_Management.png" width="880" height="750" alt="Transaction Management">
+<i>Transaction Management Diagram </i>
+</p>
 
```

Comment:
In the diagram and other places of the document, the name that refers to `TransactionMetadataTasks` is set as `Transaction Utility`. I don't know if this is the most suitable name, so maybe we can think of a better one (`TransactionManager`, `TransactionMetadataTasks`,...?). cc/ @shiveshr

Reply:
@RaulGracia  will be replaced with Transaction Management (Shivesh Suggested)!